src/plugins/data/public/ui/search_bar/search_bar.tsx

```diff
@@ -77,6 +77,8 @@ export interface SearchBarOwnProps {
   nonKqlModeHelpText?: string;
   // defines padding; use 'inPage' to avoid extra padding; use 'detached' if the searchBar appears at the very top of the view, without any wrapper
   displayStyle?: 'inPage' | 'detached';
+  // super update button background fill control
+  isFill?: boolean;
```

Comment:
I suggest we name it to make it clear that it controls fill of submit button 
maybe `fillSubmitButton`  


I think currently: `<SearchBar isFill={true} />` is not clear

Reply:
97127cff22c3041141f3c8a63f6548e9d715560d